src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -70,16 +69,10 @@ internal partial class SymbolTreeInfo : IChecksummedObject
 
         public bool ContainsExtensionMethod => _receiverTypeNameToExtensionMethodMap?.Count > 0;
 
-        private SpellChecker? _spellChecker;
-        private SpellChecker SpellChecker
-        {
-            get
-            {
-                _spellChecker ??= CreateSpellChecker(Checksum, _nodes);
```

Comment:
`SpellChecker?` was a `Nullable<SpellChecker>`.  This is just a pure struct which can have torn writes.  By moving to a `StrongBox<SpellChecker>` we have teh runtime guarantee that if `_spllChecker` is not null then `_spellChecker.Value` is 'completely written'.